exportjson.lua

```diff
@@ -172,7 +172,10 @@ do
 				end
 			end
 		end
-		local _, activityID = C_LFGList.GetActiveEntryInfo()
+		local activityEntryInfo = C_LFGList.GetActiveEntryInfo()
```

Comment:
Do we need to guard against activityEntryInfo being nil here?

Reply:
Yep, shouldn't happen, but it might avoid evoid an error in edge case